pyproject.toml

```diff
@@ -15,14 +15,18 @@ classifiers = [
   "Programming Language :: Python :: 3",
 ]
 
-requires-python = "~=3.8"
+requires-python = "~=3.9"
 dependencies = [
-  "numpy",
-  "matplotlib",
-  "colorspacious",
-  "scipy",
+  "numpy ~=1.22",
```

Comment:
Doesn't this also place an upper pin on NumPy?

Reply:
Yes, it's equivalent to `>=1.22, 1.*`, so numpy could not be upgraded to 2.x or higher.

Reply:
Ah, the setuptools docs are a bit confusing: wanted to make sure this didn't pin to 22

Reply:
Here's some good info "compatible release" specification: https://peps.python.org/pep-0440/#compatible-release